pyk/src/pyk/__main__.py

```diff
@@ -427,5 +427,44 @@ def exec_parse_outer(options: ParseOuterOptions) -> None:
         sys.stdout.write(f'{result_text}\n')
 
 
+def exec_kompilex(options: KompileXCommandOptions) -> None:
+    definition_file = Path(options.main_file).resolve()
+    search_paths = [definition_file.parent, *options.includes]
+    main_module_name = options.main_module or definition_file.stem.upper()
+
+    try:
+        final_definition = parse_outer(definition_file, main_module_name, search_paths, options.md_selector)
+    except Exception as e:
+        _LOGGER.critical(e)
+        exit(1)
```

Comment:
I think you can just let the exception propagate to the output instead.

Reply:
Sounds fine for now. There are a variety of user errors here that can cause an exception and I would eventually like to see these exceptions be collected and logged somehow for the user instead of having python die and show them a stack trace.